README.md

```diff
@@ -140,6 +140,10 @@ For `nvim-treesitter` to support a specific feature for a specific language requ
 
 The following is a list of languages for which a parser can be installed through `:TSInstall`; a checked box means that `nvim-treesitter` also contains queries at least for the `highlight` module.
 
+Experimental parsers are parsers that are maintained, but not stable enough for
+daily use yet. They are excluded from automatic installation when
+`ensure_installed` is set to `"maintained"`.
+
```

Comment:
:+1: can you add verilog to this category? You can add a comment why those parsers are experimental in `parsers.lua`. D parser has a generated grammar.js that takes 60s to compile and verilog is still uses using ABI language version 12 because their parser doesn't compile with 13.